serverless/serverless-azure-functions
# Integration tests + tag if new version

## What did you implement:

Closes #645 

Updated integration tests workflow to run node and python integration tests, but only barely - removed the tests that were trying to deploy to Azure. Node workflow also runs validation script and the node tests.

## Note:

The "Tag if new version" job runs as per Serverless framework repo here https://github.com/serverless/serverless/blob/a6bcc62182b6fe974d12d547c00e560882c55e05/.github/workflows/integrate.yml#L171.

In order for this to job to work a `USER_GITHUB_TOKEN` secret needs to be added with the 'serverless-ci' user authentication token as per Serverless framework. As per https://github.com/serverless/serverless/blob/a6bcc62182b6fe974d12d547c00e560882c55e05/.github/workflows/integrate.yml#L188-L190

## How did you implement it:

Added integration.yml in the github workflows.

## How can we verify it:

The workflow will run once the PR has been merged, but you can verify the last run in my repository here https://github.com/gligorkot/serverless-azure-functions/actions/runs/4964935025

## Todos:

_**Note: Run `npm run test:ci` to run all validation checks on proposed changes**_

       **Validate via `npm run lint`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
       **Validate via `npm test`**

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
